scripts/build/build_examples.py

```diff
@@ -131,10 +131,14 @@ def ValidateTargetNames(context, parameter, values):
     help=(
         'Set pigweed command launcher. E.g.: "--pw-command-launcher=ccache" '
         'for using ccache when building examples.'))
+@click.option(
+    '--args',
+    default="",
+    help='Additional build argument')
```

Comment:
I'm not sure whether it's the right approach with such args, but if so, this should be documented better, e.g. like where these args are passed, what is the format, etc.?